site/src/components/Resources/Resources.tsx

```diff
@@ -204,4 +204,23 @@ const useStyles = makeStyles((theme) => ({
   status: {
     whiteSpace: "nowrap",
   },
+
+  data: {
```

Comment:
I would apply the same styles from the workspace cell - how it is doing with the metadata info - but if it is already applying that, I'm good!

Reply:
This is taken straight from that component 👍